app/controllers/chargeback_controller.rb

```diff
@@ -255,8 +256,22 @@ def cb_rate_form_field_changed
     cb_rate_get_form_vars
     render :update do |page|                    # Use JS to update the display
       changed = (@edit[:new] != @edit[:current])
+      # Update the new column with the code of the currency selected by the user
+      first_new_detail = @edit[:new][:details].first
+      currency = ChargebackRateDetailCurrency.find_by(:id => first_new_detail[:currency])
+      @edit[:new][:details].each_with_index do |_detail, i|
+        new_rate_details = @edit[:new][:details][i]
+        current_rate_details = @edit[:current][:details][i]
+        next unless new_rate_details[:currency] != current_rate_details[:currency]
```

Comment:
Sorry, now that it's written like this I think maybe the double negative is hard to read...

How about:
```ruby
next if new_rate_details[:currency] == current_rate_details[:currency]
```